### PR TITLE
repo-sync-2024-01-29T16:58:35+0800

### PR DESCRIPTION
**Pull request by zhangxingmeng**
### Added
- Split learning in PyTorch supports GPU in debug mode.
- Split learning in PyTorch supports pipeline strategy.
- Sparse compression supports the compression of multi-dimensional data, such as images.
- SLModel supports input in the form of file names and allows for multiple label inputs.

- Component: prediction operator results can now save feature columns.
- Component: binning operators can now view reports directly.
- Component: binning rule modification is supported.
- Component: support model exports, support both online and offline prediction.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,52 +0,0 @@
-# See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners 
-# for more info about CODEOWNERS file
```
Comment (fy222fy) [inline .github/CODEOWNERS:2]: 这里删掉了应该不太对吧

Comment (zhouaihui) [reply .github/CODEOWNERS:2]: +1. Keep this file.

Comment (zhangxingmeng) [reply .github/CODEOWNERS:2]: 老的tools需要更新了

**Review by ian-huu — APPROVED** [review]
LGTM

**Review by da-niao-dan — APPROVED** [review]
LGTM

**Review by 6fj — APPROVED** [review]
lgtm

**Review by usafchn — APPROVED** [review]
lgtm

**Review by warriorpaw — APPROVED** [review]
LGTM

**Review by zhouaihui — APPROVED** [review]
LGTM

**Review by UniqueMarvin — APPROVED** [review]
lgtm

**Pull request merged**